dashboard/src/i18n/locales.ts

```diff
@@ -333,7 +333,7 @@ const locales0 = {
         'zh-TW': '正式',
         'en': 'Production',
         'ja': '製品',
-        'ko': '생산',
```

Comment:
You seem to have intended **Prod**,Stage,Dev   not Goods

If you're right about it, this fix is suitable